### ISSUE TITLE
loss-curves and training collapsed

### ISSUE THREAD

**Issue by yanhuanhuanyy** [junyanz/pytorch-CycleGAN-and-pix2pix#874]
@junyanz , thank you for sharing this repo.
When I train cycleGAN on my own dataset, I found that the D_A loss and D_B loss decrease and converge, but the D loss is close to 0. Is this normal or did my system collapse?
My trainA data contains 5000 images with size 160x120, and My trainB data contains 899 images with size 301x451. I use --netG resnet_9blocks, --n_layers_D 2 and 200 epoch. 
(In addition, I tried to set --n_layers_D to 3, the D loss is still close to 0.)
![D_A](https://user-images.githubusercontent.com/36001989/70721804-351e4280-1d31-11ea-8b06-8c7ea4ab2879.png)
![D_B](https://user-images.githubusercontent.com/36001989/70719117-621c2680-1d2c-11ea-823e-ecc5f9617a3e.png)
![G_A](https://user-images.githubusercontent.com/36001989/70719143-69433480-1d2c-11ea-9105-eb7463774036.png)
![G_B](https://user-images.githubusercontent.com/36001989/70719162-7102d900-1d2c-11ea-9585-952578fec921.png)
[loss_log.txt](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/files/3956097/loss_log.txt)
[train_opt.txt](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/files/3956262/train_opt.txt)

But, my training results look okay to me(I only focus on fake_B, --direction AtoB), maybe it could be better. So, is the loss curve an indicator of evaluation? Or what parameters do I need to adjust to keep the system from collapsing?
![us_syn_cyclegan images](https://user-images.githubusercontent.com/36001989/70720650-2767bd80-1d2f-11ea-8156-91b7f0791a0d.jpg)
![火狐截图_2019-12-11T09-52-09 556Z](https://user-images.githubusercontent.com/36001989/70720657-2afb4480-1d2f-11ea-8f02-835e3d172534.png)

**Comment by junyanz** [comment]
The GAN losses often oscillate and may not be the best indicator for results. You need to check the quality of results by yourself. BTW, if you want to use a discriminator with different conv layers, you need to specify `--netD n_layers` in addition to using `--n_layers_D`.

**Comment by yanhuanhuanyy** [comment]
Thank you for your quick reply. I will try to modify --netD n_layers, should I change the number of resnet's or increase the learning rate of G?

**Comment by junyanz** [comment]
Sure. Feel free to try it.

**Comment by yanhuanhuanyy** [comment]
Thanks~~
